crates/cashu/src/nuts/nut05.rs

```diff
@@ -151,6 +151,38 @@ impl<Q: Serialize + DeserializeOwned> MeltRequest<Q> {
     }
 }
 
+impl<Q: std::fmt::Display> super::nut10::SpendingConditionVerification for MeltRequest<Q> {
```

Comment:
Can you use where here https://github.com/cashubtc/cdk/blob/main/CODE_STYLE.md#generics